BREEZE.rst

```diff
@@ -482,7 +482,8 @@ running the webserver in order to log in. This can be done with the following co
 
 .. code-block:: bash
 
-    airflow users create --role Admin --username admin --password admin --email admin@example.com --firstname foo --lastname bar
+    airflow users create --email EMAIL --firstname FIRSTNAME --lastname LASTNAME --password PASSWORD --role Admin --username USERNAME
```

Comment:
Except changing the order of args, what has changed?

Reply:
I check in and no other change, did old command don't work for you? @ghost

Reply:
Oh, a deleted users......